src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
I'd recommend you do this variable non-static because static variables initialize only one time at the program's start.